_data/accessibility-tests/file-input.yml

```
@@ -0,0 +1,103 @@
+title: File input
+component_status: pass
+test_items:
+# General tests
+  - summary: File input works on mobile devices.
+    summary_additional: When you use a mobile device or tablets, file inputs should work in both portrait and landscape views with no loss of functionality.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.4
+  - summary: Color is not the only method used to convey meaning.
+    summary_additional: When you view the file input component, focus indicators, text labels and field descriptions and messaging all help establish meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.1
+  - summary: File input elements have proper contrast.
+    summary_additional: When you use a color contrast analyzer, the contrast between the file input and all of its elements is at least 4.5:1 from their respective surrounding backgrounds.
```

@amycole501, @alex-hull
**question:** Wanted to check on this one. Usually non-text elements have a 3:1 check, so wanted to see if this might have been accidentally combined with a text contrast check for 1.4.3. Should we update this check to 3:1 and also add a test for 1.4.3 to check for text contrast?

```suggestion
    summary_additional: When you use a color contrast analyzer, the contrast between the file input and all of its elements is at least 3:1 from their respective surrounding backgrounds.
```

---

@RachelCorsino 
Can you commit the suggestion above to change `4.5:1` -> `3:1`?

Additionally, can you create an additional checklist item for 1.4.3? I think this should work, but open to input!

```  
- summary: File input text meets color contrast requirements.
    summary_additional: When you use a color contrast checker to view the file input, contrast between the text color and background color is at least 4.5:1.
    test_status: pass
    test_type: general
    version_tested: 3.8.0
    wcag_criterion: 1.4.3
 ```

---

Resolved in [23c28b1](https://github.com/uswds/uswds-site/pull/2963/commits/23c28b1b772ee9faf1bc5fb99040b3d3a7fa4f45)